docs/install_guides/installing-openvino-pip.md

```diff
@@ -7,27 +7,63 @@
                  macOS operating systems, using a PyPi package.
 
 
-Using the PyPI repository, you can install either OpenVINO™ Runtime or OpenVINO Development Tools on Windows, Linux, and macOS systems.
-This article focuses on OpenVINO™ Runtime.
+.. note::
+   
+   Note that the PyPi distribution:
+   
+   * offers the Python API only
+   * does not offer support for GNA and NPU inference
+   * is dedicated to users of all major OSs: Windows, Linux, macOS
+   * may offer different hardware support under different operating systems
+     (see the drop-down below for more details)
+
+   .. dropdown:: Inference Options 
+
+      The PyPi distribution for macOS currently supports the arm64 architecture only.
```

Comment:
We activate x86_64 as well - as for example
https://files.pythonhosted.org/packages/4d/70/ac75b270b7e33a28aab12b9dec2a47e63202457285bc35c1b3454cb43229/openvino-2023.0.2-11065-cp311-cp311-macosx_10_12_x86_64.whl